### PR TITLE
fix: Doctrine ORM 3 support

### PR DESCRIPTION
**Pull request by dunglas**
`doctrine:mapping:convert`, `doctrine:ensure-production-settings` and `doctrine:mapping:import` have been removed from Doctrine 3. Registering the proxy classes prevent the container to build properly.

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -59,7 +59,11 @@
         "vimeo/psalm": "^4.7"
     },
     "config": {
-        "sort-packages": true
+        "sort-packages": true,
+        "allow-plugins": {
```
Comment (ostrolucky) [inline composer.json:63]: this is unrelated change, it needs comment from you

**Review by ostrolucky — CHANGES_REQUESTED** [review]

File: composer.json (hunk 1 above)
Comment (dunglas) [reply composer.json:63]: It's a new behavior of Composer. This is added when running `composer update`.
